app/controllers/salons_controller.rb

```diff
@@ -0,0 +1,20 @@
+class SalonsController < ApplicationController
+  def new
```

Comment:
keep new and listing separate, 

new -> new salon form
index -> listing of user

both actions are different